test_regress/t/t_vpi_public_depth_bad.pl

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env perl
+if (!$::Driver) { use FindBin; exec("$FindBin::Bin/bootstrap.pl", @ARGV, $0); die; }
+# DESCRIPTION: Verilator: Verilog Test driver/expect definition
+#
+# Copyright 2010 by Wilson Snyder. This program is free software; you
+# can redistribute it and/or modify it under the terms of either the GNU
+# Lesser General Public License Version 3 or the Perl Artistic License
+# Version 2.0.
+# SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+
+scenarios(simulator => 1);
+
+top_filename("t/t_vpi_public_depth.v");
+
```

Comment:
I'm getting "/usr/scratch/anolte/local/vlts/hrt-verilator/test_regress/t/t_vpi_public_depth.cpp:24:10: fatal error: Vt_vpi_public_depth.h: No such file or directory" Do you know how to fix this?  Am I able to resuse this .v and .cpp file? @wsnyder